plugin-server/src/worker/ingestion/hooks.ts

```diff
@@ -264,11 +266,13 @@ export class HookCommander {
         postgres: PostgresRouter,
         teamManager: TeamManager,
         organizationManager: OrganizationManager,
+        fetchHostnameGuardTeams: Set<number>,
```

Comment:
since we're going to remove it in the future and have all teams opt into this I'd add it as the last arg and optional, so there's less test changes needed when removing

Reply:
Makes sense, made optional – although didn't put it last, because we never pass `statsd` in in these tests, while we do pass this